self-service-portal/src/api/RegistryService.js

```diff
@@ -9,10 +9,10 @@ export const getParticipantSearch = async ({headers = {}}) => {
 
 export const getParticipant = async (email) => {
     var payload = { "filters": { "primary_email": { "eq":  email} } };
-    return post("/applicant/search?fields=communication,sponsors,onboard_validation_properties", payload);
+    return post("/applicant/search?fields=communication,sponsors,onboard_validation_properties,mockparticipants", payload);
```

Comment:
Try to use a variable for the below string to avoid changes in multiple places.
`communication,sponsors,onboard_validation_properties,mockparticipants`